### PR TITLE
#91 Add support for other mime types when listing blobs by prefix.

### LINKED ISSUE #91: Add support for other mime types when listing blobs by prefix.
The puzzle `77-be2897cf` from #77 has to be resolved: 

https://github.com/artipie/files-adapter/blob/ef2412c7595dbd1301f9cf01a2337a1899a651f4/src/main/java/com/artipie/files/FilesSlice.java#L59-L61

The puzzle was created by @baudoliver7 on 10-Dec-21. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by dgarus**
Closes #91

### CODE REVIEW

**Review by olenagerasimova — APPROVED** [review]
@baudoliver7 thanks

**Comment by dgarus** [comment]
@olenagerasimova you are welcome ))
thanks for the review

File: src/main/java/com/artipie/files/FilesSlice.java [hunk 1]
```diff
@@ -56,9 +56,13 @@
...
- * @todo #77:30min Add support for other mime types when listing blobs by prefix.
- *  We already add support for `text/plain`. We should also add support for mime type
- *  `text/html` and `application/json`.
+ * @todo #91:30min Add support for `text/html` mime types when listing blobs
+ *  by prefix. We already add support for `text/plain` and `application/json`.
+ *  We should also add support for mime type `text/html`.
+ *  @todo #91:30min Test FileSlice when listing blobs by prefix in JSON.
+ *  We previously introduced {@link BlobListJsonFormat}
+ *  to list blobs in JSON from a prefix. We should now test that the type
+ *  and value of response's content are correct when we make a request.
```
Comment (g4s8) [inline src/main/java/com/artipie/files/FilesSlice.java:65]: `@todo` body should start with one space indentation:
```
@todo <ticket>:<estimate> <title>
 <body>
 <body>
```

**Review by g4s8 — CHANGES_REQUESTED** [review]
@dgarus thanks, please check my comment about `@todo` formatting issue

**Review by g4s8 — APPROVED** [review]

**Pull request merged**